doc/reference/cmdline.rst

```diff
@@ -548,4 +548,158 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
+  database.
+| The :option:`--addons-path` option is required to specify the path(s) to the
+  module folder(s).
+| If combined with :option:`--path`, the count will be that of the sum of both
+  options' results (with possible overlaps). At least one of these two options is
+  required to specify which code to process.
+
+.. code-block:: console
+
+   $ odoo-bin cloc --addons-path=addons -d my_database
+
+.. seealso::
+   - :ref:`reference/cmdline/cloc/database-option`
+
+
+.. option:: -p <path>, --path <path>
+
+| Process the files in the provided path.
+| If combined with :option:`--database`, the count will be that of the sum of both
+  options' results (with possible overlaps). At least one of these two options is
+  required to specify which code to process.
+
+.. code-block:: console
+
+   $ odoo-bin cloc -p addons/account
+
+
+Multiple paths can be provided by repeating the option.
+
+.. code-block:: console
+
+   $ odoo-bin cloc -p addons/account -p addons/sale
+
+.. seealso::
+   - :ref:`reference/cmdline/cloc/path-option`
+
+
+.. option:: --addons-path <directories>
+
+| Comma-separated list of directories in which modules are stored. These directories
+  are scanned for modules.
+| Required if the :option:`--database` option is used.
+
+
+.. option:: -c <directories>
+
+Specify a configuration file to use in place of the :option:`--addons-path` option.
+
+.. code-block:: console
+
+    $ odoo-bin cloc -c config.conf -d my_database
+
+
+.. option:: -v, --verbose
+
+Show the details per file.
+
+
+Processed files
+---------------
+
+.. _reference/cmdline/cloc/database-option:
+
+With the :option:`--database` option
+''''''''''''''''''''''''''''''''''''
+
+Odoo Cloc counts the lines in each file of extra installed modules in a
+given database. In addition, it counts the python lines of server actions and
+custom computed fields stored directly in the database. In other words, that are
+not defined in any standard or importable module.
+
+Some files are excluded from the count by default:
+
+- The manifest (:file:`__manifest__.py` or :file:`__openerp__.py`)
+- The content of the folder :file:`static/lib`
+- The tests defined in the folder :file:`tests` and :file:`static/tests`
+- The xml files declared in the ``demo`` or ``demo_xml`` sections of the manifest
+
+A list of files that should be ignored by Odoo Cloc can be defined per module. This
+is specified by the ``cloc_exclude`` entry of the manifest:
+
+.. code-block:: python
+
+    "cloc_exclude": [
+        "lib/common.py", # exclude a single file
+        "data/*.xml", # exclude all xml file in a specific folder
+        "example/**/*", # exclude all file in the folder and its subfolder
+    ]
+
+| The syntax ``**/*`` can be used to ignore an entire module. This can be useful
+  to exclude a module from maintenance service costs.
+| For more information about the syntax, see `glob
+  <https://docs.python.org/3.6/library/pathlib.html#pathlib.Path.glob>`_.
```

Comment:
Perhaps don't pin the version, could even remove it entirely, docs.python.org redirects to the latest one automatically.

```suggestion
  <https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob>`_.
```